manifests/a/Anytype/Anytype/0.23.5/Anytype.Anytype.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Anytype.Anytype
+PackageVersion: 0.23.5
+PackageLocale: en-US
+Publisher: Anytype
+PackageName: Anytype
+License: Proprietary
+Copyright: Copyright © 2022 Anytype
+ShortDescription: Anytype

```

Comment:
Needs a better ShortDescription

Once you're at it you can add PublisherUrl, PackageUrl and Tags